libs/input-mapping/src/Table/Options/RewrittenInputTableOptions.php

```diff
@@ -15,15 +15,10 @@ public function __construct(array $definition, string $source, int $sourceBranch
     {
         parent::__construct($definition);
         $this->definition['source'] = $source;
-        $this->definition['sourceBranchId'] = $sourceBranchId;
+        $this->definition['source_branch_id'] = $sourceBranchId;
```

Comment:
Tohle bylo na pytel. Bylo to pojmenovane podle parametru ktery jde do SAPI (jsou camelCase), ale my mae definition snake case

Reply:
jasně